code/datums/mind.dm

```diff
@@ -76,6 +76,11 @@
 	/// your bank account id in your mind
 	var/account_id
 
+	///Assoc list of addiction values, key is the type of withdrawal (as singleton type), and the value is the amount of addiction points (as number)
+	var/list/addiction_points
+	///Assoc list of key active addictions and value amount of cycles that it has been active.
+	var/list/active_addictions
```

Comment:
**(Non blocking review)**
I don't really care too much and you can just close this review, but there isn't much of a reason to use singleton addictions here as oppossed to just creating a new addiction of a certain type when a player gains an addiction. By making addictions a singleton, we have to waste time on initialisation creating all the addiction singletons and we have to have more complex behaviours here instead of just storing the addition points in the addiction itself.